### DISCUSSION TITLE
I have been blocked because of one of my customer's fault

### DISCUSSION THREAD

**Discussion by ozgur-dogan** [vercel/community#928] [Help] [answered]
My user account has been blocked because of a customer of mine, which I provide software development service.

I have multiple organizations on my user on Vercel. As a software developer I provide development service for these companies.

One of my customer account is blocked because of their fault (they used a trademark). Instead of blocking just that organization, Vercel blocked my user completely. 

I am not able to provide my service to other client's of mine. I need immediate support.

* I am not able to login to open a support ticket and the email support is not responded yet.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @ozgur-dogan. There should be a recent response in your inbox to the case you opened via email this morning (00102047)
